templates/kotlin/base/macros/methodNeedsSecurityParameters.twig

```diff
@@ -0,0 +1,3 @@
+{% macro methodNeedsSecurityParameters(method) %}
```

Comment:
if macros are used in only one place place it in that file only. Let's keep all macros that are used more than once together as utils.twig

Reply:
okay sure .